packages/core/index.d.ts

```diff
@@ -437,7 +437,7 @@ declare module '@rjsf/core' {
 
 declare module '@rjsf/core/lib/components/fields/SchemaField' {
     import { JSONSchema7 } from 'json-schema';
-    import { FieldProps, UiSchema, IdSchema, FormValidation } from '@rjsf/core';
+    import Form, { FieldProps, UiSchema, IdSchema, FormValidation } from '@rjsf/core';
```

Comment:
Why is this import needed?

Reply:
Checking on this @mattcosta7 ?

Reply:
it's not. i'll drop that update.  I actually don't think most of these imports are needed